### PR TITLE
[ADD] purchase_order_weight_volume

### PR DESCRIPTION
**Pull request by ilyasProgrammer**
- Display weight and volume in Purchase Order Line.
- Display total order weight and total order volume in Purchase Order.

### CODE REVIEW

File: purchase_order_weight_volume/readme/DESCRIPTION.rst [hunk 1]
```diff
@@ -0,0 +1,2 @@
+- Display weight and volume in Purchase Order Line.
+- Display total order weight and total order volume in Purchase Order.
```
Comment (francesco-ooops) [inline purchase_order_weight_volume/readme/DESCRIPTION.rst]:
```suggestion
- Display total order weight and total order volume in Purchase Order, tab "Other information".
```

File: purchase_order_weight_volume/readme/USAGE.rst [hunk 2]
```diff
@@ -0,0 +1,12 @@
+To use this module, you need to:
+
+* Go to Purchase settings.
+* Enable checkboxes "Display Order Weight in PO" and/or "Display Order Weight in PO".
+* Go to Products. Update some products with weight and volume values.
+* Go to Purchase.
+* Create new RFQ. Add products.
+* You will see new column with total line items weight and volume.
+* You will see order total weight and volume.
```
Comment (francesco-ooops) [inline purchase_order_weight_volume/readme/USAGE.rst]:
```suggestion
* You will see order total weight and volume in tab "Other information".
```

**Review by francesco-ooops — APPROVED** [review]
Functional ok!

**Review by SicurSam — APPROVED** [review]
Functional ok!

File: purchase_order_weight_volume/readme/USAGE.rst [hunk 3]
```diff
@@ -0,0 +1,12 @@
+To use this module, you need to:
+
+* Go to Purchase settings.
+* Enable checkboxes "Display Order Weight in PO" and/or "Display Order Volume in PO".
+* Go to Products. Update some products with weight and volume values.
+* Go to Purchase.
+* Create new RFQ. Add products.
+* You will see new column with total line items weight and volume.
+* You will see order total weight and volume, tab "Other information".
```
Comment (francesco-ooops) [inline purchase_order_weight_volume/readme/USAGE.rst:9]:
```suggestion
* You will see order total weight and volume, tab "Other information".
* From this tab, you can also choose whether to display weight and/or volume info in report.
```

**Comment by francesco-ooops** [comment]
@OCA/procurement-purchase-maintainers is this good? :)

**Comment by francesco-ooops** [comment]
@OCA/procurement-purchase-maintainers could you please review?

**Review by HviorForgeFlow — APPROVED** [review]
Functional Review :+1:

**Pull request merged**